src/m2n/M2N.cpp

```diff
@@ -178,6 +204,29 @@ void M2N::send(double itemToSend)
   }
 }
 
+void M2N::broadcastSendLocalMesh(mesh::Mesh &mesh)
+{
+  int meshID = mesh.getID();
+  if (utils::MasterSlave::_slaveMode || utils::MasterSlave::_masterMode) {
+    assertion(_areSlavesConnected);
+    assertion(_distComs.find(meshID) != _distComs.end());
+    assertion(_distComs[meshID].get() != nullptr);    
+    _distComs[meshID]->broadcastSendMesh(mesh);
+  } else { //coupling mode
```

Comment:
Please add assertion and understandable message

Reply:
done!

Reply:
The error message makes no sense here, probably a copy-and-paste mistake.